src/components/ConversationSettings/EditableTextField.vue

```diff
@@ -279,6 +279,11 @@ export default {
 	:deep(.rich-contenteditable) {
 		flex-grow: 1;
 	}
+
+	:deep(.rich-text--wrapper) {
+		text-align: start;
```

Comment:
Why `text-align: start;` ?

Reply:
For right-to-left languages

Reply:
Updated styles for tables to not overflow the message body, and updated template for RTL support. Good to go